src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -28,6 +36,25 @@ protected PythonNodeBase(IEnumerable<PortModel> inPorts, IEnumerable<PortModel>
             ArgumentLacing = LacingStrategy.Disabled;
         }
 
+        private PythonEngineVersion engine = PythonEngineVersion.IronPython2;
+
+        [JsonConverter(typeof(StringEnumConverter))]
+        /// <summary>
+        /// Return the user selected python engine enum.
+        /// </summary>
+        public PythonEngineVersion Engine
```

Comment:
do we need a minimal test to make sure this is "covered"

Reply:
@mjkkirschner You mean serialization? I can add one

Reply:
I just mean some test that accesses this property in some way - maybe to check the default? So that the coverage report does not mark this method as uncovered.

Reply:
An explicit serialization test sounds good thuough!

Reply:
@mjkkirschner Added a serialization test to make sure coverage